reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -613,11 +613,13 @@ <h3 class="">Packages</h3>
             <td><a href="#Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0-pkg-1">1</a></td><td>Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0</td><td></td><td><em>Detected Licenses (from <a href="https://github.com/oss-review-toolkit/ort.git">VCS</a>):</em>
               <dl>
                 <dd>
-                  <div>BSD-3-Clause (<a href="https://github.com/oss-review-toolkit/ort/tree/master/analyzer/src/funTest/assets/projects/synthetic/gradle/lib/src/code.cpp#L1">link</a> to the location)</div>
+                  <div>
```

Comment:
Commit message nits:

> user experiences

I believe singular "user experience" is more common.

> The links are only supplied when ORT has the license in the
> spdx-utils resources.
> In order to have consistent links, links to the software heritage
> are used.

To me, these sentences are somehow reversed. You start with an implementation detail, and then you take up the Software Heritage topic from the title again.

How about this commit message instead:

```
For an enhanced the user experience, link license names to the respective
license texts. To avoid the use of different sources for SPDX license texts
and texts for LicenseRefs, leverage only Software Heritage for linking to
file content.

A previous idea was to link to the ORT repository itself, but that would
not work for forks of ORT.
```